graylog2-server/src/main/java/org/graylog2/bindings/ServerBindings.java

```diff
@@ -174,7 +176,7 @@ private void bindSingletons() {
         bind(ClusterStatsModule.class).asEagerSingleton();
         bind(ClusterConfigService.class).to(ClusterConfigServiceImpl.class).asEagerSingleton();
         bind(GrokPatternRegistry.class).in(Scopes.SINGLETON);
-        bind(Engine.class).toInstance(Engine.createEngine());
+        bind(Engine.class).annotatedWith(Names.named("HtmlSafe")).toProvider(HtmlSafeJmteEngineProvider.class).asEagerSingleton();
```

Comment:
Why not bind the plain Engine as a singleton too? Looks like we create a new instance with each instance of EmailSender.

Reply:
make sense. thank you :+1: